### PR TITLE
expressions,tidb: unary handes bool type

### PR DESCRIPTION
**Pull request by siddontang**
fix `select + (1 > 0), - (1 > 0)`

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
LGTM

File: expression/expressions/unary.go [hunk 1]
```diff
@@ -164,6 +164,11 @@ func (u *UnaryOperation) Eval(ctx context.Context, args map[interface{}]interfac
 		switch x := a.(type) {
 		case nil:
 			return nil, nil
+		case bool:
```
Comment (shenli) [inline expression/expressions/unary.go]: We have bool data in tidb?

Comment (siddontang) [inline expression/expressions/unary.go]: some comparison expressions return bool.

Comment (siddontang) [inline expression/expressions/unary.go]: I will change these later. but now,we must support bool.

**Comment by shenli** [comment]
LGTM

**Pull request merged**